### ISSUE TITLE
lseg demo

### ISSUE THREAD

**Issue by better07** [krrish94/lseg-minimal#1]
Thank you "lseg-minimal" for your work, I have a question about LSeg. I tried to use the LSeg demo, and manually downloaded the "demo_e200.ckpt" and the "pytorch.model.bin" from the timm, but I'm having a problem with "Missing key(s) in state_dict: "net.clip pretrained.visual.mask embedding"., what's going on please!
This is my code to creat_model:
model = timm.create_model("vit_large_patch16_384", pretrained=pretrained, pretrained_cfg_overlay=dict(file='/path/to/pytorch_model.bin'))
